omp/test/factorization/par_ilu_kernels.cpp

```diff
@@ -159,18 +186,20 @@ TEST_F(ParIlu, KernelComputeParILUIsEquivalentToRef)
     auto coo_omp = Coo::create(omp);
     csr_omp->convert_to(gko::lend(coo_omp));
     initialize_lu(&l_ref, &u_ref, &l_omp, &u_omp);
-    auto u_transpose_ref = u_ref->transpose();
-    auto u_transpose_omp = u_omp->transpose();
+    auto u_transpose_lin_op_ref = u_ref->transpose();
+    auto u_transpose_ptr_ref = static_cast<Csr *>(u_transpose_lin_op_ref.get());
+    auto u_transpose_lin_op_omp = u_omp->transpose();
+    auto u_transpose_ptr_omp = static_cast<Csr *>(u_transpose_lin_op_omp.get());
```

Comment:
Minor: maybe `csr` instead of `ptr`, since the other one is (rightfully) `lin_op`.

Reply:
That is a good idea, I think I will go with `_csr_ptr` because it is no longer an `std::unique_ptr` after the cast (and the reason I put `_ptr` in there).